spec/shoryuken/polling_spec.rb

```diff
@@ -98,3 +98,142 @@
     end
   end
 end
+
+describe Shoryuken::Polling::StrictPriority do
+  let(:queue1) { 'shoryuken' }
+  let(:queue2) { 'uppercut' }
+  let(:queue3) { 'other' }
+  let(:queues) { Array.new }
+  subject { Shoryuken::Polling::StrictPriority.new(queues) }
+
+  describe '#next_queue' do
+    it 'cycles when declared desc' do
+      # [shoryuken, 2]
+      # [uppercut,  1]
+      queues << queue1
+      queues << queue1
+      queues << queue2
+
+      expect(subject.next_queue).to eq(queue1)
+      expect(subject.next_queue).to eq(queue2)
+      expect(subject.next_queue).to eq(queue1)
+      expect(subject.next_queue).to eq(queue2)
+    end
+
+    it 'cycles when declared asc' do
```

Comment:
The priorities aren't different, the priorities are the number. They are declaration order independent.

Therefore
```
[shoryuken, 2]
[uppercut,  1]
```

and

```
[uppercut,  1]
[shoryuken, 2]
```

Have the same meaning, because they have the same priority numbers.